packages/documentation/src/content/docs/overview/concepts/accounting.mdx

```diff
@@ -136,9 +136,7 @@ You should define and adjust each peer's liquidity based on your liquidity risk
 
 <details>
 <summary>Peer liquidity example</summary>
-You and Cloud Nine Wallet are peers. You’ve agreed to extend Cloud Nine Wallet a line of credit worth \$100.00 USD. This means Cloud Nine Wallet has \$100.00 in their peer liquidity account on your Rafiki instance.
-
-Your Rafiki instance can receive packets that total up to \$100.00 to wallet addresses issued by Cloud Nine Wallet. When the \$100.00 is used up, Cloud Nine Wallet settles with you by sending \$100.00 via the shared settlement mechanism outlined in your peering agreement. When you receive the funds, you reset their liquidity in Rafiki.
+You and Cloud Nine Wallet are peers. You’ve agreed to extend Cloud Nine Wallet a line of credit worth \$100.00 USD. This means Cloud Nine Wallet has \$100.00 in their peer liquidity account on your Rafiki instance. Your Rafiki instance can receive packets that total up to \$100.00 from Cloud Nine Wallet. When the \$100.00 is used up, Cloud Nine Wallet settles with you by sending \$100.00 via the shared settlement mechanism outlined in your peering agreement. When you receive the funds, you reset their liquidity in Rafiki.
 
 </details>
```

Comment:
Just below on L153, the purpose of the wallet address payment should say for funds received via SPSP (instead of SPSP or Web Monetization)